qiskit/providers/models/backendproperties.py

```diff
@@ -83,19 +83,19 @@ def __repr__(self):
         return f"Nduv({repr(self.date)}, {self.name}, {self.unit}, {self.value})"
 
 
-class Gate:
+class GateProperties:
     """Class representing a gate's properties
```

Comment:
I renamed this (and put in a backwards-compatibility alias below) to avoid introducing an annoying Sphinx conflict in the crossreference `` :class:`.Gate` `` with `qiskit.circuit.Gate`.